aspnetcore/mvc/views/razor.md

```diff
@@ -15,74 +15,74 @@ uid: mvc/views/razor
 
 By [Taylor Mullen](https://twitter.com/ntaylormullen) and [Rick Anderson](https://twitter.com/RickAndMSFT)
 
-## What is Razor?
-
-Razor is a markup syntax for embedding server based code into web pages. The Razor syntax consists of Razor markup, C# and HTML. Files containing Razor generally have a *.cshtml* file extension.
+Razor is a markup syntax for embedding server-based code into webpages. The Razor syntax consists of Razor markup, C#, and HTML. Files containing Razor generally have a *.cshtml* file extension.
 
 ## Rendering HTML
 
-The default Razor language is HTML. Rendering HTML from Razor is no different than in an HTML file. A Razor file with the following markup:
+The default Razor language is HTML. Rendering HTML from Razor markup is no different than rendering HTML from an HTML file. If you place the following markup into a Razor file, it's rendered by the server unchanged:
 
-```html
+```cshtml
 <p>Hello World</p>
 ```
 
-Is rendered unchanged as `<p>Hello World</p>` by the server.
+```html
+<p>Hello World</p>
+```
 
 ## Razor syntax
 
-Razor supports C# and uses the `@` symbol to transition from HTML to C#. Razor evaluates C# expressions and renders them in the HTML output. Razor can transition from HTML into C# or into Razor-specific markup. When an `@` symbol is followed by a [Razor reserved keyword](#razor-reserved-keywords) it transitions into Razor-specific markup, otherwise it transitions into plain C#.
+Razor supports C# and uses the `@` symbol to transition from HTML to C#. Razor evaluates C# expressions and renders them in the HTML output.
 
-<a name=escape-at-label></a>
+When an `@` symbol is followed by a [Razor reserved keyword](#razor-reserved-keywords), it transitions into Razor-specific markup. Otherwise, it transitions into plain C#.
 
-HTML containing `@` symbols may need to be escaped with a second `@` symbol. For example:
+`@` symbols are escaped by an `@` symbol:
```

Comment:
`are` => `can be`

Reply:
@NTaylorMullen If you want an `@` symbol, do you *always* need to escape it? If so, then I'd go with "are," because "can be" infers that it's optional.

Reply:
I was thinking `can be` since you have the option of escaping it. If you don't escape it then it has a different behavior.

Reply:
I guess the broader question is: Under what circumstances does one need to escape it? Is it ok for email addresses anywhere? Are there any other scenarios where one can put a single `@` in markup (attributes or content) to get a single rendered `@` and it not :boom: on them? What I prob need to do here is explain it out a bit more.